ethers-providers/src/lib.rs

```diff
@@ -706,7 +706,7 @@ pub trait Middleware: Sync + Send + Debug {
 
     async fn fee_history(
         &self,
-        block_count: u64,
+        block_count: U256,
```

Comment:
Nit: Make this `T: Into<U256>` to make this easier to call

Reply:
Definitely, makes sense!